example/satosa/templates/qr_code.html

```diff
@@ -28,7 +28,7 @@
                             background-color: #fff;
                         "
                         >
-                        <img class="icon-qr-code" src="{{ qrcode_logo_path }}" slot="icon" />
+                        <img class="icon-qr-code" src="{{ static+qrcode_logo_path }}" slot="icon" />
```

Comment:
I don't like this solution

what happen if qrcode_logo_path doesn't start with "/"?
please provide it formatted OR add in the confgiuration validator (pydantic) the check that all the configuration to be appended to the static webendpoint MUST start with "/"